demisto_sdk/commands/postman_codegen/tests/postman_codegen_test.py

```diff
@@ -204,24 +205,56 @@ def test_duplicate_requests_check_duplicates_dont_exist(self):
         }
         duplicate_requests_check(names_dict)
 
+    test_files_path = os.path.join(git_path(), 'demisto_sdk', 'commands', 'postman_codegen', 'tests', 'test_files')
+    with open(os.path.join(test_files_path, 'shared_args_path.json')) as shared_args_path_file:
+        shared_args_path_items = json.load(shared_args_path_file)
+
+    more_then_one_paths = defaultdict(int, {'name': 2, 'description': 3, 'url': 3, 'method': 3})
+    one_for_each_paths = defaultdict(int, {'name': 1, 'id': 1, 'uid': 1})
+    complicated_chars_paths = defaultdict(int, {'name': 1, 'required': 5})
+    SHARED_ARGS_PATH = ((shared_args_path_items['more_then_one_items'], more_then_one_paths),
+                        (shared_args_path_items['no_path_items'], defaultdict(int)),
+                        (shared_args_path_items['one_for_each_items'], one_for_each_paths),
+                        (shared_args_path_items['complicated_chars_items'], complicated_chars_paths))
+
+    @pytest.mark.parametrize('flattened_json, shared_args_path', SHARED_ARGS_PATH)
+    def test_find_shared_args_path(self, flattened_json, shared_args_path):
```

Comment:
Separate the test-case when there's no duplicated items to another unittest

Reply:
There are no duplicated items between the test cases here (they are just all in the same json file, but refer different items in it).